Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/NodeGraphWidget.h

```diff
@@ -60,7 +60,9 @@ namespace EMStudio
 
         QPoint LocalToGlobal(const QPoint& inPoint) const;
         QPoint GlobalToLocal(const QPoint& inPoint) const;
-        QPoint SnapLocalToGrid(const QPoint& inPoint, uint32 cellSize = 10) const;
+
+        static constexpr uint32 s_snapCellSize = 10;
```

Comment:
nit: consider adding a comment to what this is, how was set (if known) ie what should future maintainers know about this const?